clion/src/main/kotlin/org/rust/clion/profiler/perf/RsPerfConfigurationExtension.kt

```diff
@@ -45,6 +46,7 @@ class RsPerfConfigurationExtension : CargoCommandConfigurationExtension() {
         val project = configuration.project
         val settings = CPPProfilerSettings.instance.state
         val perfPath = settings.executablePath.orEmpty()
+        validatePerfSettings(project)
```

Comment:
Maybe it's worth to make similar validation for DTrace as well?
See `com.intellij.profiler.clion.DTraceProfilerConfigurationExtensionKt#validateDTraceSettings` in CLion code base and it's usage

Reply:
Good idea, I'll do it in the next PR

Reply:
@Undin Done https://github.com/intellij-rust/intellij-rust/pull/6427. What do you think about putting DTrace settings validation to the current release, together with Perf settings validation?

Reply:
I don't see any urgent reason to cherry-pick #6427 into the release branch

Reply:
OK